mubench.reviewsite/tests/create_test_database.php

```diff
@@ -50,24 +67,26 @@
 Schema::create($finding->getTable(), function(Blueprint $table) {
     $table->increments('id');
     $table->integer('experiment_id');
+    $table->integer('misuse_id');
     $table->string('project_id', 30);
     $table->string('version_id', 30);
-    $table->string('misuse_id', 30);
+    $table->string('misuse', 30);
```

Comment:
This is very confusing, because this is still called the misuse id in some places. I understand that we have the primary-key ids and the MuBench ids, but we should use a consistent naming scheme. How about calling the primary keys id (as by convention) and the MUBench ids `mid` or `mubid` (going for pronounceable here)?